storage/persistent_storage/persistent-storage-aws.adoc

```diff
@@ -26,13 +26,19 @@ storage provider.
 
 .Additional References
 
+* xref:../../storage/container_storage_interface/persistent-storage-csi-ebs.adoc#persistent-storage-csi-ebs[AWS Elastic Block Store CSI Driver Operator]
 * link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html[Amazon
 EC2]
 
 // Defining attributes required by the next module
 :StorageClass: EBS
 :Provisioner: kubernetes.io/aws-ebs
 
+:FeatureName: AWS EBS CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+0]
```

Comment:
This doesn't look right in preview build. Because we can't do an include in a module, we should manually add text in the Operator Install section of this topic.